### PR TITLE
Implement Ser Ilyn Payne

### PR DESCRIPTION
**Pull request by DukeTax**
Not sure if card.getCost() is a thing.

### CODE REVIEW

File: server/game/cards/characters/02/serilynpayne.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const DrawCard = require('../../../drawcard.js');
+
+class SerIlynPayne extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Kneel Ser Ilyn Payne to kill a character with printed cost 3 or less',
```
Comment (cryogen) [inline server/game/cards/characters/02/serilynpayne.js]: Try to condense this, it's a little wordy and wont look so great on the menu

File: server/game/cards/characters/02/serilynpayne.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            method: 'killChud'
```
Comment (cryogen) [inline server/game/cards/characters/02/serilynpayne.js]: can we call this something better? :)

**Comment by cryogen** [comment]
The spacing is a little off in this file.  Are you using an editor that supports eslint? If not run eslint manually on the source to find issues like this

**Comment by DukeTax** [comment]
I have just been typing directly on here so far, but I'm using an editor with eslint now. :)

**Pull request merged**